.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: java
+jdk: openjdk8
+sudo: required
+dist: trusty
+
+install:
+- wget http://www-us.apache.org/dist/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
+- tar xzf apache-maven-3.3.9-bin.tar.gz
+- curl -s -L https://github.com/mozilla/smarthome/releases/latest | egrep -o /mozilla/smarthome/releases/download/.*/m2repository.tar.gz | wget --base=http://github.com/ -i -
+- tar xf m2repository.tar.gz -C $HOME/.m2/repository/
+
+before_script:
+- rm $HOME/.m2/settings.xml
+- export MAVEN_OPTS="-Dmaven.artifact.threads=3 -Xmx2469m -XX:MaxPermSize=512m"
+- export PROJECTS="--projects !extensions/voice/org.eclipse.smarthome.voice.mactts/pom.xml"
```

Comment:
So you didn't change the ``TTSServiceMacOS.xml`` to only run on mac? You should do this.

This is a bug upstream[[1]](https://github.com/eclipse/smarthome/blob/master/extensions/voice/org.eclipse.smarthome.voice.mactts/OSGI-INF/TTSServiceMacOS.xml#L15) that I've fixed downstream[[2] ](https://github.com/mozilla/smarthome/blob/master/bundles/io/org.eclipse.smarthome.io.voice/OSGI-INF/TTSServiceMacOS.xml#L16)and we shouldn't re-introduce their bug.